src/components/Onboarding/Screens/Apps/IssuanceSettings/IssuanceChart.js

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from 'react'
+import { ChartBase } from '@components/Onboarding/kit/ChartComponents'
+import { ResponsiveLine } from '@nivo/line'
+import { useCharts } from '@/providers/Charts'
+import { generateElements } from '@/utils/conviction-modelling-helpers'
+
+const DEFAULT_MAX_MONTH = 12
+const DEFAULT_INCREMENT = 12 / 365
+
+const calculateRatio = (
+  initialRatio,
+  targetRatio = 0.001,
+  maxAdjustmentRatioPerYear
+) => {
+  let balance = initialRatio
+  let supply = 1
+  const ratio = [balance / supply]
+  for (let i = 0; i < 365; i++) {
```

Comment:
A small comment with an explanation of what is happening here would be apprecited. I kind of understand, thought seems a piece of logic worth giving more context about it.